common/lib/xmodule/xmodule/x_module.py

```diff
@@ -398,7 +398,7 @@ def get_content_titles(self):
         the display_names of all its leaf nodes.
         """
         if self.has_children:
-            return sum((child.get_content_titles() for child in self.get_children()), [])
+            return sum((child.get_content_titles() for child in self.get_display_items()), [])
```

Comment:
That seems pretty arbitrary. @cpennington, do you think get_children should be consistent about whether or not access is checked?